packages/api-electron/index.js

```diff
@@ -55,6 +55,28 @@ module.exports = (url) => {
     });
   });
 
+  ipc.on('message-service', (e, msg) => {
```

Comment:
Should probably prefix all of the events with `ssf-`

Maybe describe what the event is with the event name, like: `ssf-send-message`?